tests/clients/test_opening_explorer.py

```diff
@@ -34,3 +34,44 @@ def test_correct_rating_params(self):
                 json={},
             )
             Client().opening_explorer.get_lichess_games(ratings=["1200", "1400"])
+
+
+class TestMasterGames:
+    @pytest.mark.vcr
+    def test_result(self):
+        res = Client().opening_explorer.get_masters_games(
+            play=["d2d4", "d7d5", "c2c4", "c7c6", "c4d5"]
+        )
+        assert res["white"] == 1667
+        assert res["black"] == 1300
+        assert res["draws"] == 4428
+
+
+class TestPlayerGames:
+    @pytest.mark.vcr
+    @pytest.mark.default_cassette("TestPlayerGames.results.yaml")
+    def test_wait_for_last_results(self):
+        result = Client().opening_explorer.get_player_games(
+            player="evachesss", color="white", wait=True
+        )
+        assert result["white"] == 125
+        assert result["draws"] == 18
+        assert result["black"] == 133
```

Comment:
Hey, is there any particular reason to check for the exact number of games? This make the test brittle, and I don't think it tests anything more than just sending the requests.

Reply:
It tests that the response is correct, e.g. really waits until the indexing is finished. It's true that hard-coding numbers mean you need to adjust them if you record new response mocks but that shouldn't really be necessary so seems fine imo.